.travis.yml

```diff
@@ -4,28 +4,40 @@ sudo: false
 matrix:
     include:
         - python: "2.7"
-          env: TOXENV=docs,lint-py27
+          env: TOXENV=docs,lint
         - python: "3.6"
-          env: TOXENV=docs,lint-py36
+          env: TOXENV=docs,lint
+        - python: "2.6"
+          env: TOXENV=py26,datetime
         - python: "2.7"
-          env: TOXENV=py27
-        - python: "2.7"
-          env: TOXENV=py27-datetime
+          env: TOXENV=py27,datetime
+        - python: "3.4"
+          env: TOXENV=py34,datetime
```

Comment:
Python 3.4 is no longer supported since RestrictedPython 4.0b4. I see no need for another maintenance burden by testing it.